pkg/cmd/server/admin/create_kubeconfig.go

```diff
@@ -111,6 +124,9 @@ func (o CreateKubeConfigOptions) Validate(args []string) error {
 	if len(o.UserNick) == 0 {
 		return errors.New("user-nick must be provided")
 	}
+	if len(o.ContextNamespace) == 0 {
```

Comment:
required from the command line, but defaults when composed from another command?